code/modules/antagonists/traitor/equipment/Malf_Modules.dm

```diff
@@ -176,71 +176,85 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
 	if (owner_AI.shunted)
 		return //prevent AI from activating doomsday while shunted, fucking abusers
 	active = TRUE
-	set_us_up_the_bomb(owner)
+	set_up_us_the_bomb(owner)
 
-/datum/action/innate/ai/nuke_station/proc/set_us_up_the_bomb(mob/living/owner)
-	var/pass = prob(10) ? "******" : "hunter2"
+/datum/action/innate/ai/nuke_station/proc/set_up_us_the_bomb(mob/living/owner)
```

Comment:
Could you explain the name change?

Reply:
Matches the reference. Figured I might as well fix it while I'm here.

Reply:
Oh, I see, I missed the reference. 👍